test/com/m3/octoparts/util/OctoMetricsImplSpec.scala

```diff
@@ -0,0 +1,44 @@
+package com.m3.octoparts.util
+
+import com.codahale.metrics.JvmAttributeGaugeSet
+import com.codahale.metrics.jvm.{ GarbageCollectorMetricSet, MemoryUsageGaugeSet, ThreadStatesGaugeSet }
+import com.m3.octoparts.support.PlayAppSupport
+import org.scalatest.{ BeforeAndAfterEach, Matchers, FunSpec }
+import scala.collection.JavaConverters._
+
+class OctoMetricsImplSpec extends FunSpec with Matchers with PlayAppSupport with BeforeAndAfterEach {
+
+  override protected def afterEach(): Unit = {
+    subject.onStop()
+  }
+
+  lazy val subject = new OctoMetricsImpl(appComponents.applicationLifecycle, appComponents.configuration)
+
+  describe("#onStart") {
+
+    it("should not throw") {
+      subject.onStart()
+    }
+
+    it("should work with consecutive calls") {
```

Comment:
I'm bordering on pedantry, but does "should work" mean "should not throw" in this context?